Sources/Sentry/SentryProfiler.mm

```diff
@@ -181,14 +191,11 @@ @implementation SentryProfiler {
 
 + (void)initialize
 {
-#    if SENTRY_TARGET_PROFILING_SUPPORTED
```

Comment:
We had a ton of these, but the entire file was also wrapped in one starting on line 3, so i removed all the intermediary ones.